tools/flaky-test-reporter/json_reporter.go

```diff
@@ -55,7 +56,7 @@ func writeFlakyTestsToJSON(repoDataAll []RepoData, dryrun bool) error {
 			for test := range testSet {
 				testList = append(testList, test)
 			}
-			if err := run(
+			if err := helpers.Run(
```

Comment:
Since you are using the helpers, can we delete these two files from here?

Reply:
I did delete them. Github diff "intelligently" considered them as renames :-(

Reply:
oh. okies